wolfssh/internal.h

```diff
@@ -483,8 +483,8 @@ typedef struct HandshakeInfo {
 #endif
 
     byte useEcc;
-#ifndef WOLFSSH_NO_SABER_LEVEL1_SHA256
-    byte useSaber:1;
+#ifndef WOLFSSH_NO_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256
+    byte useEccKyber;
```

Comment:
Did you mean to remove the `:1;`?

Reply:
Yes.  similar to useEcc this will eventually be used to show which level we will be supporting (ie 1, 3, 5) so I wanted to take out the `:1` ASAP .